test/e2e/create_test.go

```diff
@@ -116,7 +116,7 @@ func (tc *testContext) waitForWindowsNodes(nodeCount int32, waitForAnnotations,
 				log.Printf("waiting for %d Windows nodes", gc.numberOfNodes)
 				return false, nil
 			}
-			return false, err
+			return false, nil
```

Comment:
I suggest logging the errors here too. It will be useful for us to see the different sort of transient errors we are seeing.